examples/Applepay/index.php

```diff
@@ -0,0 +1,240 @@
+            cursor: not-allowed;
+            line-height: 1.5;
+        }
+
+        .unsupportedBrowserMessage p {
+            margin: 0;
+        }
+
+        .applePayButtonContainer {
+            position: relative;
+        }
+    </style>
+</head>
+
+<body style="margin: 70px 70px 0;">
+
+<p><a href="https://docs.unzer.com/docs/testdata" target="_blank">Click here to open our test data in new tab.</a></p>
+
+<form id="payment-form" class="unzerUI form" novalidate>
+    <!-- This is just for the example - Start -->
+    <div class="fields inline">
+        <label for="transaction_type">Chose the transaction type you want to test:</label>
+        <div class="field">
+            <div class="unzerUI radio checkbox">
+                <input type="radio" name="transaction_type" value="authorize" checked="">
+                <label>Authorize</label>
+            </div>
+        </div>
+        <div class="field">
+            <div class="unzerUI radio checkbox">
+                <input type="radio" name="transaction_type" value="charge">
+                <label>Charge</label>
+            </div>
+        </div>
+    </div>
+    <!-- This is just for the example - End -->
+
+    <div>
+        <div class="field" id="error-holder" style="color: #9f3a38"> </div>
+        <div class="button-well">
+            <div class="applePayButtonContainer">
+                <div class="apple-pay-button apple-pay-button-black" lang="us"
+                     onclick="setupApplePaySession()"
+                     title="Start Apple Pay" role="link" tabindex="0"></div>
+            </div>
+        </div>
+    </div>
+</form>
+
+<script>
+    const $errorHolder = $('#error-holder');
+
+    // Create an Unzer instance with your public key
+    let unzerInstance = new unzer('<?php echo UNZER_PAPI_PUBLIC_KEY; ?>');
+
+    const APPLE_PAY_VERSION = 6;
+
+    if (!window.ApplePaySession || !ApplePaySession.canMakePayments() || !ApplePaySession.supportsVersion(APPLE_PAY_VERSION)) {
+        handleError('This device does not support Apple Pay version 6!', APPLE_PAY_VERSION);
+    } else {
+        var unzerApplePayInstance = unzerInstance.ApplePay();
+
+        var applePayInformationObject = unzerApplePayInstance.createApplePaySessionRequest({
+            countryCode: 'DE',
+            currencyCode: 'EUR',
+            supportedNetworks: ['visa', 'masterCard'],
+            merchantCapabilities: ['supports3DS', 'supportsCredit', 'supportsDebit'],
+            totalLabel: '"Unzer UI Demo (Card is not charged)"',
+            totalAmount: '0.0',
+
+            onMerchantValidationCallback: (session, event) => {
+                $.post('./merchantvalidation.php', JSON.stringify({"merchantValidationUrl": event.validationURL}), null, 'json')
+                    .done(function (validationResponse) {
+                        // Contains Applepay session on succsess.
+                        session.completeMerchantValidation(validationResponse);
+                    })
```

Comment:
I think "merchant session" ist the right word here (not Applepay session)
see here: https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession/1778021-onvalidatemerchant

Reply:
changed it to "applepay merchant session" to make clear its from appleay.